Mathlib/Topology/Order/Category/FrameAdjunction.lean

```diff
@@ -41,7 +41,7 @@ open CategoryTheory Order Set Topology TopologicalSpace
 
 namespace Locale
 
-/-! ### Definition of the points functor `pt` --/
+/-! ### Definition of the points functor `pt` - -/
```

Comment:
Would you like to search for all `- -/` occurrences?

Reply:
I think that there are no more of these, other than commented out comments/doc-strings and `/-- -/` docstrings [sic].